sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/model/ErrorReport.java

```diff
@@ -80,22 +80,43 @@ public void setThreadName(String threadName) {
         this.threadName = threadName;
     }
 
+    /**
+     * Gets the stack trace of the crash.
+     *
+     * @return The stack trace.
+     */
+    public String getStackTrace() {
+        return stackTrace;
+    }
+
+    /**
+     * Sets the stack trace of the crash.
+     *
+     * @param stackTrace The stack trace.
+     */
+    public void setStackTrace(String stackTrace) {
+        this.stackTrace = stackTrace;
+    }
+
     /**
      * Gets the throwable.
      *
+     * @deprecated This method has been deprecated, use {@link #getStackTrace()} instead.
      * @return The throwable.
      */
+    @Deprecated
     public Throwable getThrowable() {
-        return throwable;
+        return null;
     }
 
     /**
      * Sets the throwable.
      *
+     * @deprecated This method has been deprecated, use {@link #setStackTrace(String)} instead.
```

Comment:
The setter was public just because ErrorLogHelper is sitting in a different package and lack of internal keyword in Java but was not meant to be part of the public API surface.

We could remove the method.